package.json

```diff
@@ -25,13 +25,13 @@
     "url": "https://phabricator.wikimedia.org/project/profile/1771/"
   },
   "dependencies": {
-    "commander": "~2.12.2"
+    "commander": "~4.1.1"
   },
   "devDependencies": {
-    "coveralls": "^3.0.0",
+    "coveralls": "^3.0.9",
     "istanbul": "^0.4.5",
```

Comment:
Same, maybe not a part of this patch, but Istanbul was deprecated. We're using nyc now. This PR has an example how to move to it: https://github.com/wikimedia/mediawiki-title/pull/40/files

I'm ok if it's not a part of this update, just general comment.

Reply:
I would prefer to do it later. Recorded as https://phabricator.wikimedia.org/T246011